odxtools/parameters/systemparameter.py

```diff
@@ -1,19 +1,25 @@
 # SPDX-License-Identifier: MIT
+import getpass
 from dataclasses import dataclass
+from datetime import datetime
 from typing import List, Optional
 from xml.etree import ElementTree
 
 from typing_extensions import override
 
-from ..decodestate import DecodeState
 from ..encodestate import EncodeState
-from ..exceptions import odxrequire
+from ..exceptions import odxraise, odxrequire
 from ..odxlink import OdxDocFragment
 from ..odxtypes import ParameterValue
 from ..utils import dataclass_fields_asdict
 from .parameter import ParameterType
 from .parameterwithdop import ParameterWithDOP
 
+PREDEFINED_SYSPARAM_VALUES = [
+    "TIMESTAMP", "SECOND", "MINUTE", "HOUR", "TIMEZONE", "DAY", "WEEK", "MONTH", "YEAR", "CENTURY",
+    "TESTERID", "USERID"
```

Comment:
Where did this list come from?

Reply:
they are listed in table 5 on page 80 of the ASAM spec for ODX 2.2.

Reply:
Maybe add a comment for reference?

Reply:
fair point. fixed: https://github.com/mercedes-benz/odxtools/pull/347/commits/61c25a8d88c01df37082a9b9f5da3aa3742d8c9f